authentication/models.py

```diff
@@ -4,6 +4,8 @@
 
 class AppUser(AbstractUser):
     is_verified_user = models.BooleanField(default=False)
+    real_username = models.TextField(null=True)
```

Comment:
I don't think the new props is needed. Since this will just create redundancy in our code. I think you can change the logic instead.

Reply:
I agree. If we change the model, we'll need to manually add those values for every existing user in our production database. Instead of modifying the model, why not create the AppUser instance when verification succeeds?

Reply:
No, to create the token WE NEED THE USER OBJECT, that's why the user needs to bw created first.

Reply:
Oh, okay thanks for the clarification. Why not check the `is_verified boolean` first before this code block:
```
return Response({'msg': 'Username is already taken by a verified user!'}, status=400)
```
This won't require you to change the model.

Reply:
Ok, will try